src/lib/api/api.go

```diff
@@ -577,7 +602,8 @@ func (a *API) marshalled() []byte {
 	return js
 }
 
-func (a *API) send(w http.ResponseWriter, data []byte) {
+func (a *API) send(w http.ResponseWriter, data []byte, message string) {
+	a.marshalled()
```

Comment:
something like this over here
```suggestion
	data := marshalled(status, data)
```